securedrop/tests/pages-layout/test_journalist.py

```diff
@@ -104,13 +104,13 @@ def test_admin_new_user_two_factor_hotp(self):
         self._admin_logs_in()
         self._admin_visits_admin_interface()
         self._admin_creates_a_user(hotp='123456')
-        self._screenshot('journalist-admin_new_user_two_factor_hotp.png')
+        self._screenshot('journalist-admin.new_user_two_factor_hotp.png')
 
     def test_admin_new_user_two_factor_totp(self):
         self._admin_logs_in()
         self._admin_visits_admin_interface()
         self._admin_creates_a_user(hotp=None)
-        self._screenshot('journalist-admin_new_user_two_factor_totp.png')
+        self._screenshot('journalist-admin.new_user_two_factor_totp.png')
```

Comment:
Let's not rename any screenshots - this will break the screenshot autoupdating functionality